subnets.go

```diff
@@ -24,15 +25,42 @@ type SubnetStub struct {
 	UUID string `json:"uuid,omitempty"`
 }
 
+type SubnetCreateRequest struct {
+	ZonalResourceRequest
```

Comment:
This is not a zonal resource. The zone is only part of the network.

Reply:
done